### ISSUE TITLE
2021-4-23

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#522]
171Load a DGL-provided dataset; Build a GNN model with DGL-provided neural network modules; Train and evaluate a GNN model for node classification on either CPU or GPU; (2021-3-29)
172What is ground truth? 就是「标准答案」的意思。(2021-3-29)
173What is 半监督学习? 什么是半监督学习？既有有标记数据 xr，又有无标记数据 xu，一般无标记数据的数量远大于有标记数据。半监督学习又可以分为两种：Transductive learning:无标记数据就是Testing data. Inductive learning:无标记数据不是 testing data，假设在训练时不知道 testing set. 为什么要用半监督学习（Semi-supervised learning）？因为收集数据比较容易，但是收集label数据的代价却很昂贵。(2021-3-29)
174Overview of Node Classification with GNN; GNNs, by contrast, offers an opportunity to obtain node representations by combining the connectivity and features of a local neighborhood; With the help of only a small portion of labeled nodes, a graph neural network (GNN) can accurately predict the node category of the others; (2021-3-29)
175Add classify with GCN to paper; done; (2021-3-29)
176微信看一看广告文章过滤技术浅析 https://hub.baai.ac.cn/view/7324 (2021-3-30)
177资源推荐 |《智能计算系统》在线课程，陈云霁授课 https://hub.baai.ac.cn/view/7313 (2021-3-30)
178Self-supervised learning is a means for training computers to do tasks without humans providing labeled data (i.e., a picture of a dog accompanied by the label “dog”).  (2021-3-30)
179预训练 | 自监督预训练提升了自监督预训练 https://hub.baai.ac.cn/view/7321 (2021-3-30)
